modules/33acrossBidAdapter.js

```diff
@@ -267,10 +307,66 @@ function _buildBannerORTB(bidRequest) {
     ? _getViewability(element, utils.getWindowTop(), minSize)
     : NON_MEASURABLE;
 
+  const ext = contributeViewability(viewabilityAmount);
+
   return {
     format,
-    ext: contributeViewability(viewabilityAmount)
+    ext
+  }
+}
+
+// BUILD REQUESTS: VIDEO
+// eslint-disable-next-line no-unused-vars
+function _buildVideoORTB(bidRequest) {
+  const videoAdUnit = utils.deepAccess(bidRequest, 'mediaTypes.video', {});
+  const videoBidderParams = utils.deepAccess(bidRequest, 'params.video', {});
+
+  const video = {}
+
+  const sizes = videoAdUnit.playerSize || [];
+  const {w, h} = _getSize(sizes);
+  video.w = w;
+  video.h = h;
+
+  // Obtain all ORTB params related video from Ad Unit
+  VIDEO_ORTB_PARAMS.forEach((param) => {
+    if (videoAdUnit.hasOwnProperty(param)) {
+      video[param] = videoAdUnit[param];
+    }
+
+    // Override with bidder specific video params
+    if (videoBidderParams.hasOwnProperty(param)) {
+      video[param] = videoBidderParams[param];
+    }
+  });
+
+  const product = _getProduct(bidRequest);
+
+  // Placement Inference Rules:
+  // - If no placement is defined then default to 2 (In Banner)
+  // - If product is instream (for instream context) then override placement to 1
+  video.placement = video.placement || 2;
+
+  if (product === PRODUCT.INSTREAM) {
+    video.startdelay = video.startdelay || 0;
```

Comment:
still, the startdelay can be negative number, try to use isNaN

Reply:
A negative number is not falsy, is it? Although the above must check for non-numbers for sure. So will add NaN check, definitely.

Reply:
Actually, check for non-numbers not required as this is covered during validation phase. If placement or startdelay is set and is not a number then request fails validation. Since validation happens before buildRequest it is safe to assume that the type of params passed is as expected.

Reply:
hmm, my bad, the negative number is not falsy, then we do not to check NaN